src/D2L.Bmx/Program.cs

```diff
@@ -217,10 +217,14 @@
 		await UpdateChecker.CheckForUpdatesAsync( configProvider.GetConfiguration() );
 		//Creating new Cache directory and removing old files.
 		if( !Directory.Exists( BmxPaths.CACHE_DIR ) ) {
-			Directory.CreateDirectory( BmxPaths.CACHE_DIR );
-			File.Delete( Path.Join( BmxPaths.BMX_DIR, "awsCredsCache" ) );
-			File.Delete( Path.Join( BmxPaths.BMX_DIR, "sessions" ) );
-			File.Delete( Path.Join( BmxPaths.BMX_DIR, "update_check" ) );
+			try {
+				Directory.CreateDirectory( BmxPaths.CACHE_DIR );
+				File.Delete( Path.Join( BmxPaths.BMX_DIR, "awsCredsCache" ) );
+				File.Delete( Path.Join( BmxPaths.BMX_DIR, "sessions" ) );
+				File.Delete( Path.Join( BmxPaths.BMX_DIR, "update_check" ) );
+			} catch ( Exception ex ) {
+				Console.WriteLine( "Error cleaning up old files continuing..." );
```

Comment:
``Console.Error.Write`` here, we pipe the output from stdout when using bmx like: ``bmx print | Invoke-Expression``
So any text that needs to be shown but not passed into piped commands needs to go to stderr.